specification/src/main/asciidoc/platform/BrandMapping.adoc

```diff
@@ -0,0 +1,114 @@
+[cols=2, options=header]
+.Java EE to Jakarta EE Mapping
+|===
+|Java EE Technology
+|Jakarta EE Technology
+
+|Java™ Platform, Enterprise Edition 8
+|Jakarta™ EE Platform 8
+
+|Enterprise JavaBeans™ 3.2
+|Jakarta™ Enterprise Beans 3.2
+
+|JavaServer Pages™ 2.3
+|Jakarta™ Server Pages 2.3
+
+|Expression Language 3.0
+|Jakarta™ Expression Language 3.0
+
+|Common Annotations for the Java Platform 1.3
+|Jakarta™ Annotations 1.3
+
+|Java™ Servlet 4.0
+|Jakarta™ Servlet 4.0
+
+|Java™ Message Service 2.0
+|Jakarta™ Messaging 2.0
+
+|Java™ Transaction API 1.2
+|Jakarta™ Transaction 1.3
+
+|JavaMail™ API 1.6
+|Jakarta™ Mail 1.6
+
+|JavaBeans™ Activation Framework 1.1
+|Jakarta™ Activation 1.2
+
+|Java EE™ Connector Architecture 1.7
+|Jakarta™ Connectors 1.7
+
+|Web Services for Java EE 1.4
+|Jakarta™ XML Web Services 2.3
+
+|Java™ API for XML-based RPC 1.1
+|Jakarta™ XML RPC 1.1
+
+|SOAP with Attachments API for Java™ 1.3
```

Comment:
Should be 1.4.

Reply:
Java EE 8 specifies SOAP 1.3...

Reply:
Correct. The Java EE 8 specification references (correctly or not) version 1.3.  We have version 1.4.  I do not have an explanation so put a TODO in that section. Merging and completing this is probably the best approach.